packages/gatsby-source-wordpress/src/steps/create-schema-customization/build-types.js

```diff
@@ -47,38 +48,24 @@ const unionType = typeBuilderApi => {
 }
 
 const interfaceType = typeBuilderApi => {
-  const { type, schema, gatsbyNodeTypes, fieldAliases, fieldBlacklist } =
-    typeBuilderApi
+  const { type, schema } = typeBuilderApi
 
   const state = store.getState()
-  const { ingestibles, typeMap } = state.remoteSchema
+  const { ingestibles } = state.remoteSchema
   const { nodeInterfaceTypes } = ingestibles
 
-  const allTypes = typeMap.values()
-
-  const implementingTypes = Array.from(allTypes)
-    .filter(
-      ({ interfaces }) =>
-        interfaces &&
-        // find types that implement this interface type
-        interfaces.find(singleInterface => singleInterface.name === type.name)
-    )
-    .map(type => typeMap.get(type.name))
-    .filter(
-      type =>
-        type.kind !== `UNION` ||
-        // if this is a union type, make sure the union type has one or more member types, otherwise schema customization will throw an error
-        (!!type.possibleTypes && !!type.possibleTypes.length)
-    )
+  const implementingTypes = getTypesThatImplementInterfaceType(type)
 
```

Comment:
I abstracted all this code into `getTypesThatImplementInterfaceType` because it needs to be run during query generation as well as schema customization, where before it was only needed during schema customization.